overlord/hookstate/ctlcmd/mount.go

```diff
@@ -219,20 +218,10 @@ func (m *mountCommand) Execute([]string) error {
 	}
 
 	sysd := systemd.New(systemd.SystemMode, nil)
-	name, err := m.createMountUnit(sysd)
+	_, err = m.createMountUnit(sysd)
```

Comment:
Maybe this should be called instead `createMount`, or `createAndStartMountUnit`?

Reply:
Well, it is more than that. It is created, enabled, and either reloaded, restarted or started.

Reply:
Maybe `persistantlyMount`. Or just `ensureMount` (because it also does nothing if it is already there and it is already mounted).

Reply:
Probably `ensureMount` is the most consistent with what we have, +1 on that one.

Reply:
Yeah, I like ensureMount as well!